db/schema.rb

```diff
@@ -197,6 +200,7 @@
     t.string "item_file_name", limit: 255
     t.integer "item_file_size", limit: 4
     t.datetime "item_updated_at"
+    t.string "token", limit: 255
```

Comment:
I know the db schema is kind of borked but can you please remove all the other changes from this branch so that it's just the addition of token.